sqs/src/main/scala/akka/stream/alpakka/sqs/javadsl/package.scala

```diff
@@ -0,0 +1,12 @@
+/*
+ * Copyright (C) 2016-2017 Lightbend Inc. <http://www.lightbend.com>
+ */
+package akka.stream.alpakka.sqs
+
+import akka.japi.function.Function
+
+package object javadsl {
+  def identityFunction[T] = new Function[T, T] {
+    override def apply(param: T): T = param
+  }
```

Comment:
Isn't used anywhere so can be removed? (If it was used a `val` would be better, to avoid the allocation of a new function on every use)